src/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```diff
@@ -229,4 +229,4 @@
     <TargetingPackReference Include="System.Core" />
   </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
Did this file change at all other than whitespace?  Looks like it didn't, in which case can you just revert this change?